packages/react/src/contexts/DefaultOptionsContexts.ts

```diff
@@ -2,8 +2,9 @@ import { createContext } from 'react'
 import type { DelayProps, SuspenseProps } from '..'
 import type { OmitKeyOf } from '../utility-types'
 
-export const DelayDefaultPropsContext = createContext<OmitKeyOf<DelayProps, 'children'>>({
+export const DelayDefaultPropsContext = createContext<DelayProps>({
   ms: undefined,
+  fallback: undefined,
```

Comment:
```ts
const defaultProps = useContext(DelayDefaultPropsContext)
  
const fallback = typeof props.fallback === 'undefined' ? defaultProps.fallback : props.fallback
```

Since we use `defaultProps.fallback` as above, what do you think about defining fallback for clarity?

`SuspenseDefaultPropsContext` already has a fallback defined. 

```tsx
export const SuspenseDefaultPropsContext = createContext<OmitKeyOf<SuspenseProps, 'children' | 'devMode'>>({
  fallback: undefined,
  clientOnly: undefined,
})
```

Reply:
Cool!